src/containers/WelcomePage/FrontpageSubjects.jsx

```diff
@@ -159,10 +160,20 @@ class FrontpageSubjects extends Component {
       this.state.preview,
     );
 
-    const allSubjects = config.isNdlaProdEnvironment
+    let allSubjects = config.isNdlaProdEnvironment
       ? frontpageCategories
       : [...frontpageCategories, getAllImportSubjectsCategory(subjects)];
-
+    allSubjects = allSubjects.map(i => {
```

Comment:
Vet ikke om jeg helt liker at man assigner til samme array som man mapper. Er det noen grunn til å for eksempel ikke putte det i en egen funksjon som gjør det vi vil? Da er det lettere å teste også.